packages/freezed/lib/src/templates/from_json_template.dart

```diff
@@ -35,11 +37,19 @@ class FromJson {
         ''';
       }).join();
 
+      var defaultCase = 'throw FallThroughError();';
+      if (constructors.any((element) => element.isFallback)) {
```

Comment:
Rather than `any`, do the `singleWhere` query before the if, and do `fallbackConstructor != null`. To avoid iterating over the list twice.

Reply:
done